src/plugins/facebook.ts

```diff
@@ -235,8 +235,8 @@ export class Facebook {
   @Cordova()
   static logEvent(
     name: string,
-    params?: Object,
-    valueToSum?: number
+    params: Object = null,
```

Comment:
Adding default values wont make a difference since the method will be overwritten by the decorator.